pages/planetusermanual.md

```diff
@@ -32,3 +32,72 @@ Under the PLANET system, there are two types of structures: **Nations** and **Co
   * A **Community** is not able to connect to other **Communities**.
 
 The **nations** may store thousands of Resources in many formats and languages.  Unique collections of Resources can be selected and downloaded into each **Community** from the **Nation**.  This gives a Learner a personalized approach to his or her education.
+
+## How do I navigate and use PLANET?
+###_How do I access PLANET and become a member?_
+
+1. To create an account use a web browser to navigate to your Nation or Community and click on “_Become a Member_” to get started.
+
+![](uploads/images/PlanetNavigateMemberLogin.png)
+
+2. Fill out the resulting form and completing the form, click “_Register_.” You are now a member!
+
+![](uploads/images/planetnavigateAddmember.png)
+
+NOTE: Each PLANET system is unique; universal memberships do not exist.  If you become a member of a specific Nation or Community, you do not automatically become a member of any other existing Nations and Community.  To become a member of another Nation or Community you must register with a new account.
+
+###_What are the differences between Learners, Leaders, and Managers?_
+In the PLANET system, there are three different kinds of users: **Learners**, **Leaders**, and **Managers**.
+
+**Learner** is the default user level when a person first joins. Each person continues to be a Learner regardless of additional responsibilities and roles. All Learners have access to standard and individualized user functionality.
+
+**Leaders** are Learners that have created a Course or have become _Course Leaders_. Leaders have access to all Learner functionalities and to additional Course management functionalities.
+
+**Managers** have access to Learner, Leader, and additional functionalities that allow them to manage local Resources, publications, collections, surveys, data sync schedules, and local software updates.  
```

Comment:
> In order to see the available manager functionalities, click on “Manager”

In the google doc `Manager` is also italic.